Eatery/Views/Eateries/FilterBar.swift

```diff
@@ -145,10 +145,41 @@ class FilterBar: UIView {
 
     @objc func buttonPressed(sender: UIButton) {
         sender.isSelected.toggle()
-
+        
         if sender.isSelected {
             let filter = displayedFilters[sender.tag]
             selectedFilters.insert(filter)
+
+            // check if filters match that of campus or ctown (taking into account
+            // that nearest first is shared by both)
+            let isCTown = displayedFilters.allSatisfy({ displayedFilter -> Bool in
```

Comment:
Instead of having this be done in the filter bar (where we don't necessarily know whether the filters are for Ctown or campus), we can move this into the EateriesVC subclasses.